primefaces/src/main/resources/org/primefaces/Messages_es.properties

```diff
@@ -15,4 +15,4 @@ primefaces.password.INVALID_MATCH = {0}: Error de validación.
 primefaces.password.INVALID_MATCH_detail = {0} debería coincidir con {1}.
 primefaces.slider.OUT_OF_RANGE = {0}: Error de validación: el valor está fuera de rango.
 primefaces.tree.REQUIRED = {0}: Error de validación.
-primefaces.tree.REQUIRED_detail = Se requiere selección.
\ No newline at end of file
+primefaces.tree.REQUIRED_detail = Selección obligatoria.
```

Comment:
I need to see the context of this translation but I think maybe can be better: "La selección es requerida."

Reply:
In Spanish/castellan, the word `requerida` isn't usually used. Only in technical context. This message is for user. The best option is to use `Selección obligatoria`.